lxd/cluster/connect.go

```diff
@@ -112,7 +112,7 @@ func SetupTrust(cert, targetAddress, targetCert, targetPassword string) error {
 	post.Type = "client"
 	err = target.CreateCertificate(post)
 	if err != nil && err.Error() != "Certificate already in trust store" {
-		return errors.Wrap(err, "failed client cert to cluster")
+		return errors.Wrap(err, "failed to add client cert to cluster")
```

Comment:
Errors must start with a capital (and never end with a period)

Reply:
I realize this one was already there, but might as well fix it since you're changing it :)

Reply:
Fixed.